packages/nimble-components/src/table-column/extensions/fractional-width-column.ts

```diff
@@ -0,0 +1,46 @@
+import { attr, nullableNumberConverter } from '@microsoft/fast-element';
+import type { TableColumn } from '../base';
+
+export interface FractionalWidthColumn {
+    fractionalWidth: number;
+    disableResize: boolean;
+    minPixelWidth: number | null;
+}
+
+// prettier-ignore
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function fractionalWidthColumn<TBase extends abstract new (...args: any[]) => TableColumn>(base: TBase): TBase {
+    /**
+     * The Mixin that provides a concrete column with the API to support being resized
+     * proportionally within a Table.
+     */
+    abstract class FractionalWidthColumn extends base implements FractionalWidthColumn {
+        public fractionalWidth = 1;
+
+        public disableResize = false;
+
+        public minPixelWidth: number | null = null;
+
+        public fractionalWidthChanged(): void {
+            this.internalFractionalWidth = this.fractionalWidth;
+        }
+
+        public minPixelWidthChanged(): void {
+            if (this.minPixelWidth !== null) {
+                this.internalMinPixelWidth = this.minPixelWidth;
```

Comment:
I don't think it's the correct behavior to ignore changes to `minPixelWidth` when its value isn't a number. Instead, I think you need to reset it back to its default value.

For example, if a client removes the `min-pixel-width` attribute, I wouldn't expect the column to maintain the most recent valid configuration of `min-pixel-width`. Instead, I would expect the column to behave as though `min-pixel-width` had never been set.